DomainContent/Zaru/applause-app/applause.html

```diff
@@ -0,0 +1,241 @@
+    <link href="https://fonts.googleapis.com/css?family=Raleway:300,400,600,700" rel="stylesheet">
+    <style>
+        body {
+            margin: 10;
+            width: 100% - 10px;
+            font-family: 'Raleway', sans-serif;
+            color: white;
+            background: linear-gradient(#2b2b2b, #0f212e);
+        }
+
+        .top-bar {
+            height: 90px;
+            background: linear-gradient(#2b2b2b, #1e1e1e);
+            font-weight: bold;
+            padding-left: 30px;
+            padding-right: 30px;
+            display: flex;
+            align-items: center;
+            position: fixed;
+            width: 480px;
+            top: 0;
+            z-index: 1;
+        }
+
+        .content {
+            margin-top: 90px;
+            padding: 30px;
+        }
+
+        input[type=button] {
+            font-family: 'Raleway';
+            font-weight: bold;
+            font-size: 13px;
+            text-transform: uppercase;
+            vertical-align: top;
+            height: 28px;
+            min-width: 120px;
+            padding: 0px 18px;
+            margin-right: 6px;
+            border-radius: 5px;
+            border: none;
+            color: #fff;
+            background-color: #000;
+            background: linear-gradient(#343434 20%, #000 100%);
+            cursor: pointer;
+        }
+
+        input[type=button].white {
+            color: #121212;
+            background-color: #afafaf;
+            background: linear-gradient(#fff 20%, #afafaf 100%);
+        }
+
+        input[type=button]:enabled:hover {
+            background: linear-gradient(#000, #000);
+            border: none;
+        }
+
+        input[type=button].white:enabled:hover {
+            background: linear-gradient(#fff, #fff);
+            border: none;
+        }
+
+        input[type=button]:active {
+            background: linear-gradient(#343434, #343434);
+        }
+
+        input[type=button].white:active {
+            background: linear-gradient(#afafaf, #afafaf);
+        }
+
+        input[type=button]:disabled {
+            color: #252525;
+            background: linear-gradient(#575757 20%, #252525 100%);
+        }
+
+        input[type=button][pressed=pressed] {
+            color: #00b4ef;
+        }
+
+        /* The switch - the box around the slider */
+
+        .switch {
+            position: relative;
+            display: inline-block;
+            width: 60px;
+            height: 34px;
+        }
+
+        /* Hide default HTML checkbox */
+
+        .switch input {
+            display: none;
+        }
+
+        /* The slider */
+
+        .slider {
+            position: absolute;
+            cursor: pointer;
+            top: 0;
+            left: 0;
+            right: 0;
+            bottom: 0;
+            background-color: #ccc;
+            -webkit-transition: .4s;
+            transition: .4s;
+        }
+
```

Comment:
cool slider action

Reply:
W3Schools for the win! https://www.w3schools.com/howto/howto_css_switch.asp